misc/helm-charts/operator/values.yaml

```diff
@@ -200,45 +201,45 @@ operator:
   # Node selector to use for the operator pod
   nodeSelector: {}
   resources:
-    # Resources requested by the operator for CPU and memory
+    # -- Resources requested by the operator for CPU and memory
     requests:
       cpu: 100m
       memory: 512Mi
-    # Resource limits for the operator's CPU and memory
+    # -- Resource limits for the operator's CPU and memory
     limits:
       memory: 512Mi
 
 environmentd:
-  # Node selector to use for environmentd pods spawned by the operator
+  # -- Node selector to use for environmentd pods spawned by the operator
   nodeSelector: {}
 
 clusterd:
-  # Node selector to use for clusterd pods spawned by the operator
+  # -- Node selector to use for clusterd pods spawned by the operator
   nodeSelector: {}
 
 # RBAC (Role-Based Access Control) settings
 rbac:
-  # Whether to create necessary RBAC roles and bindings
+  # -- Whether to create necessary RBAC roles and bindings
   create: true
 
 # Service account settings
 serviceAccount:
-  # Whether to create a new service account for the operator
+  # -- Whether to create a new service account for the operator
   create: true
-  # The name of the service account to be created
+  # -- The name of the service account to be created
   name: "orchestratord"
 
 # Observability settings (disabled in this case)
 observability:
   enabled: true
```

Comment:
Does this need:

# -- Whether to enable observability

Reply:
Thank you for the review Kay 🙏  Sorry for the delay on updating the PR, somehow I've missed the notifications here.

Reply:
Absolutely no worries.  Thank you for doing all the work! ❤️